### ISSUE TITLE
Open JDK does not install

### ISSUE THREAD

**Issue by GuruTom** [InsolvencyService/LaptopToolKit#3]
Open JDK will not install and produces an error

**Comment by GuruTom** [comment]
Error message:

```
TASK [java-jdk : Upgrade Packages] ****************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Could not find aptitude. Please ensure it is installed."}

```

There are issues open about this on the Ansible repository...

**Comment by GuruTom** [comment]
The current workaround is using: 

```
- name: Update apt-get
    shell: sudo apt-get update
```

**Comment by GuruTom** [comment]
Fixed in #9
